visualization/app/codeCharta/ui/customConfigs/customConfigs.component.html

```diff
@@ -1,87 +1,89 @@
-<md-menu class="custom-configs-component">
-	<md-button class="custom-configs-button" ng-click="$ctrl.initView(); $mdMenu.open($event);" title="Open your saved Custom Configs">
-		<i class="fa fa-cogs"></i>
-	</md-button>
+<div class="custom-configs">
+	<md-menu class="custom-configs-component">
+		<md-button class="custom-configs-button" ng-click="$ctrl.initView(); $mdMenu.open($event);" title="Open your saved Custom Configs">
+			<i class="fa fa-cogs"></i>
+		</md-button>
```

Comment:
Could you check your prettier configuration, cause the whole file seems to be reformatted.
Or was it incorrect before?

Reply:
It was wrong before. I checked out the branch locally and run prettier but nothing changed.

Reply:
a new div has been added.